UPDATING_BADGES_AND_PR_CHECKS.md

```diff
@@ -0,0 +1,120 @@
+# Updating badges and setting up pull request checks <!-- omit in toc -->
+
+* [Updating the badges](#updating-the-badges)
+  * [GitHub Actions build statuses](#github-actions-build-statuses)
+  * [BetterCodeHub and LGTM badges](#bettercodehub-and-lgtm-badges)
+* [Enabling pull request checks](#enabling-pull-request-checks)
+  * [BetterCodeHub](#bettercodehub)
+  * [LGTM](#lgtm)
+
+The badges at the top of [README.md](README.md) need to be modified
+
+* Every time GitHub Classroom creates a new repo from our starter code
+* Every time a group forks or copies a repository from one iteration to another
+
+Essentially anytime a new repository is created, then the badge
+URLs will need to be updated. If they are not, then the badges
+will reflect the state of the "parent" repository and not the
+current repository. This isn't difficult, but it does need to be done and should probably one of the very first "bookkeeping" tasks a team does when setting up their repository.
+
+You will also need to make sure that these services are running checks
+whenever you do pull requests. You want to make sure that you're getting
+automated checks on every pull request so that you can avoid merging
+in checks that would in some way harm the quality of the codebase.
+
+Checklist:
+
+* [ ] Fix BetterCodeHub badge
+* [ ] Fix LGTM badge(s)
+* [ ] Enable PR checks for BetterCodeHub
+* [ ] Enable PR checks for LGTM
+
+## Updating the badges
+
+Below are instructions for each of the different types of badges.
+
+### GitHub Actions build statuses
+
+These badges do not need any changes. The URLs for those badges are
+relative to your particular repository (that's what all the `../..` bits
+do), so they "automagically" work when your repo is created.
+
+Unfortunately the other badges are going from sources outside of GitHub,
+so we can't use the same trick for them.
+
+### BetterCodeHub and LGTM badges
+
+When GitHub Classroom makes the initial copies of this repository, the
+[BetterCodeHub](https://bettercodehub.com) (BCH) and
+[LGTM](https://lgtm.com) ("looks good to me") badges have several copies
+of this string:
+
+```html
```

Comment:
It _could_ be seen as (part of) an HTML document. 😜 

Markdown lint is fussy if you don't specify a language type, and that seemed the best option if I was going to provide one. This could be a case of the tail wagging the dog, though, and maybe we just remove the language spec from these fence-blocked regions. How much does it really bug you?

Reply:
I'm really not a fan of the markdown lint I guess. These strings are just that, plain strings.

Reply:
I think you can put `string` instead of `html` and the linters will decide that's "OK", and MD engines will just ignore it, so that's an option.

Reply:
Actually, it's `text` not `string`.